docs/getting-started/fullstack-example.md

```diff
@@ -24,10 +24,10 @@ KintoHub provides a dedicated Postgres server with 1 Gb of storage.
 4. Enter or generate your `Username`, `Password`, and `Root Password`.
 5. Click **Deploy** At the top right.
 
-This process will take about **1 Minute** to deploy.
+This process will take around **1 Minute** to deploy.
```

Comment:
> This will take around **1 Minute** to complete.

As below. 
Not sure if it's my English, but when I read it, I feel you are deploying a process.

Reply:
Both work for me, but ill change it.

Reply:
Also I only noticed "around" not "to complete" in your original ask.